apps/galaxy/src/modules/Orders/OpenOrders/OpenOrdersList.tsx

```diff
@@ -42,16 +67,44 @@ const OpenOrdersList = () => {
     }),
   ];
 
+  const Submit = () => {
+    const Orders = checkedItems
+      .map((checked, index) => {
+        if (checked) return index;
+      })
+      .filter((checked) => checked !== undefined);
+    const orderIds = lists
+      ?.map((checkedOrders: Order, index: number) => {
+        const status = Orders.find((val) => val == index);
+        if (status != undefined) {
+          return checkedOrders.id;
+        }
+      })
+      .filter((value: number | undefined) => value);
```

Comment:
And as discussed, this can be simplified to below

```ts

(() => {
  const lists = [{ id: 1 }, { id: 2 }, { id: 3 }, { id: 4 }, { id: 5 }, { id: 6 }];
  const checkedItems = [true, false, true, true, false, true];

  const selectedOrderIds = lists
    .filter((_item, index) => checkedItems.at(index))
    .map((item) => item.id);

  selectedOrderIds; //?
})();
```